io-engine/src/lvm/error.rs

```diff
@@ -0,0 +1,52 @@
+use snafu::Snafu;
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("Failed to parse: {err}"))]
+    FailedParsing { err: String },
+    #[snafu(display("Failed to execute command: {err}"))]
+    FailedExec { err: String },
+    #[snafu(display("I/O error: {source}"))]
+    Io { source: std::io::Error },
+    #[snafu(display("Invalid PoolType: {value}"))]
+    InvalidPoolType { value: i32 },
+    #[snafu(display("Volume Group with {query} not found"))]
```

Comment:
`Volume Group`  --> `LVM Volume Group` just to be clear it's not related to a Volume

Reply:
Done